releases/release-5.0.4.md

```diff
@@ -0,0 +1,169 @@
+    - Fix the issue that concurrently truncating the same partition hangs DDL. [#26238](https://github.com/pingcap/tidb/pull/26238)
+
++ TiKV
+
+    - RaftStore Snapshot GC fix: fix the issue that snapshot GC missed GC snapshot files when there's one snapshot file failed to be GC-ed. [#10872](https://github.com/tikv/tikv/pull/10872)
+    - Fix TiKV panic when enable Titan and upgrade from pre-5.0 version [#10843](https://github.com/tikv/tikv/pull/10843)
+    - Fix newer TiKV can't rollback to 5.0.x [#10843](https://github.com/tikv/tikv/pull/10843)
+    - Fix TiKV panic when Titan is enabled and upgrade from < 5.0 versions to >= 5.0 versions. A cluster may hit the issue if it was upgraded from TiKV 3.x and enabled Titan before the upgrade in the past. [#10778](https://github.com/tikv/tikv/pull/10778)
+    - Fix the resolve failures caused by the left pessimisic locks. [#10654](https://github.com/tikv/tikv/pull/10654)
+    - Fix duration calculation panics on certain platforms [#10571](https://github.com/tikv/tikv/pull/10571)
+    - Fix unencoded keys of `batch_get_command` in load-base-split [#10564](https://github.com/tikv/tikv/pull/10564)
+
++ TiFlash
+
+    - Fix the potential panic issue that occurs when running table scan tasks
+    - Fix the potential memory leak issue that occurs when executing `MPP` tasks
+    - Fix a bug that TiFlash raises the error about `duplicated region` when handling `DAQ` requests
+    - Fix the issue of unexpected results when executing the aggregation functions `COUNT` or `COUNT DISTINCT`
+    - Fix the potential panic issue that occurs when executing `MPP` tasks
+    - Fix a potential bug that TiFlash cannot restore data when deployed on multiple disks
+    - Fix the potential panic issue that occurs when deconstructing `SharedQueryBlockInputStream`
+    - Fix the potential panic issue that occurs when deconstructing `MPPTask`
+    - Fix the issue of unexpected results when TiFlash failed to establish `MPP` connections
+    - Fix the potential panic issue that occurs when resolving locks
+    - Fix the issue that store size in metrics is inaccurate under heavy writing
+    - Fix a bug of incorrect results that occurs when queries contain filters like `CONSTANT` `<` | `<=` | `>` | `>=` `COLUMN`
+    - Fix the potential issue that TiFlash cannot GC the delta data after running for a long time
+    - Fix a potential bug that metrics display wrong value
+    - Fix the potential issue of data inconsistency that occurs when deployed on multiple disks
+
++ PD
+
+    - Fix the bug that PD would not fix down-peer in time. [#4082](https://github.com/tikv/pd/pull/4082)
+    - Fix an issue where data is not stored when using max-replicas or location-labels to indirectly update default placement rule [#3914](https://github.com/tikv/pd/pull/3914)
+    - Fix the bug that PD may panic during scaling out TiKV. [#3910](https://github.com/tikv/pd/pull/3910)
+
```

Comment:
- Reduce the conflict due to multiple scheduler running in same time [#3856](https://github.com/tikv/pd/pull/3856)
    - Fix the issue that the scheduler may appear again even if we have already executed the delete operation [#3823](https://github.com/tikv/pd/pull/3823)